pyfar/dsp/__init__.py

```diff
@@ -37,6 +37,7 @@
 __all__ = [
     'fft',
     'filter',
+    'utils',
```

Comment:
Why do we need to add it here, since utils is in the parent dir?

Reply:
Thanks for noting. That was a leftover